client/src/main/scala/com/github/dapperware/slack/api/SlackChats.scala

```diff
@@ -7,6 +7,9 @@ import io.circe.syntax._
 import zio.ZIO
 
 trait SlackChats {
+  def permalink(channelId: String, ts: String): ZIO[SlackEnv, SlackError, String]                                   =
+    sendM(request("chat.getPermalink", "channel" -> channelId, "ts" -> ts)) >>= as[String]("permalink")
```

Comment:
In the docs the argument is called `message_ts`.

Reply:
Fixed, thanks!